app/src/main/java/com/uwaterloo/watodo/TaskAdapter.java

```diff
@@ -47,7 +50,8 @@ public void onBindViewHolder(@NonNull TaskHolder holder, int position) {
         holder.textViewTitle.setText(currentTask.getTitle());
 //        holder.textViewDescription.setText(currentTask.getDescription());
         holder.textViewPriority.setText(String.valueOf(currentTask.getPriority()));
-        holder.textViewDeadline.setText(currentTask.getDeadline());
+        String ddl = currentTask.getDdlYear() + "." + currentTask.getDdlMonth() + "." + currentTask.getDdlDay();
```

Comment:
Currently, if a task has no date set the date is displayed as "0.0.0" on the list of tasks.

Maybe we should display no string at all in the date area instead of "0.0.0"?

You can do this here in the code to achieve that:
```
int ddlYear = currentTask.getDdlYear();
int ddlMonth = currentTask.getDdlMonth();
int ddlDay = currentTask.getDdlDay();
String ddl = "";
if (ddlYear != 0 && ddlMonth != 0 && ddlDay != 0) {
	ddl = currentTask.getDdlYear() + "." + currentTask.getDdlMonth() + "." + currentTask.getDdlDay();
}
holder.textViewDeadline.setText(ddl);
```